mode/examples/Demos/Performance/Esfera.pyde

```diff
@@ -1,54 +1,51 @@
 """
-Esfera by David Pena.    
+Esfera by David Pena.
 
-ES - Distribucion aleatoria uniforme sobre la superficie de una esfera. 
+ES - Distribucion aleatoria uniforme sobre la superficie de una esfera.
 EN - Uniform random distribution on the surface of a sphere.
 """
 
 CUANTOS = 16000  # how many
-LISTA = []  # list 
+LISTA = []  # list
 RADIO = 200  # radius
 rx = 0
 ry = 0
 
-def setup() :
+
+def setup():
     size(1024, 768, P3D)
     global RADIO, LISTA
     RADIO = height/3.5
```

Comment:
I would have expected spaces around the division operator. What did you use to format?